lib/ts/recipe/thirdparty/types.ts

```diff
@@ -24,12 +24,14 @@ import {
 } from "../recipeModule/types";
 import OverrideableBuilder from "supertokens-js-override";
 
-export type PreAndPostAPIHookAction = "GET_AUTHORISATION_URL" | "THIRD_PARTY_SIGN_IN_UP";
+export type PreAndPostAPIHookAction = "GET_AUTHORISATION_URL" | "THIRD_PARTY_SIGN_IN_UP" | "GET_PROVIDERS";
 
 export type PreAPIHookContext = RecipePreAPIHookContext<PreAndPostAPIHookAction>;
 export type PostAPIHookContext = RecipePostAPIHookContext<PreAndPostAPIHookAction>;
 
 export type UserInput = {
+    clientType?: string;
+    getTenantId?: () => Promise<string | undefined>;
```

Comment:
move getTenantId into the relevant functions -> getAuthorisationUrl, getProviders

Reply:
Done